File: MODUL.5/PRAK504_2310817120010_NurHikmah.c
#include <stdio.h>
int reverse(int angka)
{
    // Lengkapi Function ini
    int reversed = 0;
    while (angka > 0)
    {
        int digit = angka % 10;
        reversed = reversed * 10 + digit;
        angka /= 10;
    }
    return reversed;
}
int main()
{
    int A, B;
    scanf("%d %d", &A, &B);
    A = reverse(A);
    B = reverse(B);
    int C = A + B;
    printf("%d", reverse(C));
}

File: MODUL.5/PRAK504_2310817120010_NurHikmah.py
def reverse(angka):
    reversed = 0
    while angka > 0:
        digit = angka % 10
        reversed = (reversed * 10) + digit
        angka //= 10
    return reversed

A, B = map(int, input().split())
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))
